Gopkg.toml

```diff
@@ -66,7 +66,7 @@ required = [
 [[constraint]]
   name = "github.com/RHsyseng/console-cr-form"
   # revision for branch "master"
-  revision = "658ef3680a1118e5822a169d8fde2567b1e04f46"
+  branch = "integration"
```

Comment:
This temporarily points to the integration branch until the corresponding [PR 32](https://github.com/RHsyseng/console-cr-form/pull/32) can be merged. It will then be a one-line change, plus generated files, to use the latest from the master branch.